.github/workflows/code_review.yaml

```diff
@@ -0,0 +1,50 @@
+# This workflow will perform the code review on each code change.
+# It uses "actions/cache" and turborepo in order to perform tasks only on changed files.
+
+name: "Code Review"
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - apps/**
+      - packages/**
+  pull_request:
+    paths:
+      - apps/**
+      - packages/**
+  workflow_dispatch: {}
+
+jobs:
+  code_review:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          fetch-depth: 2
```

Comment:
**question:** Why `fetch-depth` is set to `2`?

Reply:
To be honest, I didn't think about it, because I took the workflow from the [dx repository](https://github.com/pagopa/dx/blob/dd97ba46ae92d9be580df63dbbd836e1b4b5216b/.github/workflows/code-review.yaml#L22-L25).

But, looking the documentation, I think the value `1` would be enough for our use case.

Reply:
I agree @lucacavallaro, Indeed I'm going to remove the input variable. 😄

Reply:
I've updated the workflow accordingly to the changes made in the dx repository (using [this workflow](https://github.com/pagopa/dx/blob/main/.github/workflows/js_code_review.yaml))